locksmith/src/models/index.js

```diff
@@ -6,20 +6,23 @@ const Sequelize = require('sequelize')
 
 const basename = path.basename(__filename)
 const env = process.env.NODE_ENV || 'development'
-const config = require(__dirname + '/../config/config.js')[env]
+const config = require('../../config/config')[env]
+
 const db = {}
 
 let sequelize
-if (config.use_env_variable) {
```

Comment:
Did you intend to remove this with this shift? IIRC this file was generated

Reply:
Yes it was intentional... because we do not use this settings?
I had no idea the file was generated?